doc/howto/format2/building_msgs.rst

```diff
@@ -8,10 +8,11 @@ package.xml
 :::::::::::
 
 Your ``package.xml`` must declare a ``<build_depend>`` on
-``message_generation``, and a ``<exec_depend>`` on
-``message_runtime``::
+``message_generation``, and a ``<exec_depend>`` as well as
+``<build_export_depend>`` on ``message_runtime``::
```

Comment:
Can you switch the order to be consistent with the following xml: `<build_export_depend> as well as <exec_depend>`